src/actions/pause/fetchExtra.js

```diff
@@ -11,11 +11,7 @@ import type { ThunkArgs } from "../types";
 export function fetchExtra() {
```

Comment:
FetchExtra??? This feels a bit vague, down the line this might be really confusing. i feel like this can be much clearer.

Reply:
can you file an issue for renaming this

Reply:
Ok will do.